addons/web_editor/static/lib/odoo-editor/test/spec/color.test.js

```diff
@@ -9,20 +9,20 @@ const setColor = (color, mode) => {
 describe('applyColor', () => {
     it('should apply a color to a slice of text in a span in a font', async () => {
         await testEditor(BasicEditor, {
-            contentBefore: '<p>a<font>b<span>c[def]g</span>h</font>i</p>',
+            contentBefore: '<p>a<font odoo-test-ignore-attr>b<span odoo-test-ignore-attr>c[def]g</span>h</font>i</p>',
```

Comment:
@dmo-odoo I put a comment in the test utils to explain this. Since we sanitize at the beginning of the tests, if we want to test with attributeless spans and fonts, they're going to need an attribute that we can ignore for the tests so this is what it is.